docs/v6-migration.md

```diff
@@ -0,0 +1,103 @@
+---
+id: v6-migration
+title: v6 Migration Guide
+position: 0
+---
+
+## Overview
+
+There have been some dramatic and breaking changes made in the bump to version 6.0.0 of `cross-seed`. These changes are mostly things you will not have to do anything to realize the benefits of, while others require addressing in both your configuration file (`config.js`) as well as possibly permissions or volume/mounting if you'd like to take advantage of them.
+
+Here I will go over the changes made and any actions you may need to take to both get `cross-seed` running as well as take advantage of the new features and capabilities.
+
+:::warning HEADS-UP!
+If you are not using at least version 20 of Node, this is the first thing you will have to update to get cross-seed running. If you are using Docker you will not need to change anything regarding this, as we've already built this into the container for you.
+:::
+
+:::tip
+The general recommendation is to grab the new `config.template.js` and simply go through and migrate your settings over. This will give you the latest descriptions/comments for your reference, as well as all of the updated options with our recommended defaults.
+:::
+
+### Zod (Config.js Validation)

```

Comment:
I'd move non-breaking changes/feature either to separate page or somewhere in the end here.
ZOD is non-breaking, but linking is, or might be, so should be on top

Reply:
I like this being here, because it's going to quickly point out that I should be checking the output for detailed problem solving, possibly proving an answer before I need to read anything else.